editor/package.json

```diff
@@ -165,6 +165,7 @@
     "react-virtualized-auto-sizer": "1.0.2",
     "react-vtree": "2.0.0",
     "react-window": "1.8.5",
+    "recoil": "0.1.2",
```

Comment:
Might be worth mentioning this and talking it through a bit more than sneaking it in via a PR!